accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/authorizableutils/impl/AuthorizableCreatorServiceImpl.java

```diff
@@ -662,10 +662,7 @@ private Authorizable createNewUser(
         } else {
             newUser = userManager.createUser(principalId, password, new PrincipalImpl(principalId), intermediatePath);
         }
-
-        if (!isSystemUser) {
```

Comment:
There is no reason to not run setAuthorizableName() for system users - the current code did run this method on second run anyway (second run=when the system user already exists)